wwwapp/settings.py

```diff
@@ -72,8 +72,11 @@
     'compressor',
     'wwwapp',
     'django_cleanup',
+    'social_django',
 )
 
+SOCIAL_AUTH_POSTGRES_JSONFIELD = True
```

Comment:
This makes us strictly dependent on postgres, doesn't it? :/

Reply:
Aren't we using postgress on prod as the settings suggests?

Reply:
Yes, but code should be sqlite compatible and unless there is a very good reason, database agnostic

Reply:
It works fine in sqlite.